test/unit/gateways/worldpay_test.rb

```diff
@@ -649,6 +649,11 @@ def test_failed_verify
     assert_failure response
   end
 
+  def test_empty_inst_id_is_stripped
+    @gateway.expects(:ssl_post).with { |_, request, _| request !~ /installationId/ }.returns(successful_authorize_response)
```

Comment:
TIL

Reply:
Tx Dilan.

Not the clearest... If you know a better way, I take it! I wanted to use something like
```
with(anything, regexp_matches(/NOT installationId/), anything)
``` 
but did not find a way to inject a regular expression that would match if a string is NOT present. The 
[Regexp doc](https://ruby-doc.org/core-2.5.1/Regexp.html) has `Negative lookahead` etc., but it was even more cryptic, and I did not make work anyway :-).


I made sure the test failed before the fix.

Reply:
@bbraschi It's not required, but if you'd like to implement this in a cleaner way, I would recommend using the `stub_comms do` combined with `assert_not_match`. You can find examples of this approach in other unit tests in this file for reference.